robyn/__init__.py

```diff
@@ -48,7 +47,6 @@ def __init__(
         self.config = config
         self.dependencies = dependencies
 
-        load_vars(project_root=directory_path)
```

Comment:
@VishnuSanal , this is needed here only. We would still want to allow the variables being set in `python app.py`

Reply:
@sansyrox `cli#run` gets called before `Robyn#init`. we cannot access the env vars from `cli#run` if the env vars are loaded later!

Reply:
@VishnuSanal , that is okay. But how would you load vars when using `python app.py` ?